HIP/hip-1064.md

```diff
@@ -0,0 +1,227 @@
+---
+hip: 1064
+title: Daily Rewards For Active Nodes
+author: Neeharika Sompalli <@Neeharika-Sompalli>
+working-group: Leemon Baird (@lbaird), Richard Bair <@rbair23>, Michael Tinker <@tinker-michaelj>
+requested-by: Hashgraph
+type: Standards Track
+category: Core
+needs-tsc-approval: Yes
+status: Draft
+last-call-date-time: 2024-04-08T07:00:00Z
+created: 2024-10-17
+discussions-to: https://github.com/hashgraph/hedera-improvement-proposal/discussions/1063
+updated: 2024-10-17
+---
+
+## Abstract
+
+This HIP proposes a reward mechanism that will incentivize nodes to remain active on the network. The definition of
+active node is described below. The minimum reward per day for nodes will be a configurable property decided by the
+governance of the deployed network. This minimum reward may be adjusted based on fees already paid throughout the day.
+
+### Terminology
+
+<dl>
+<dt>Judge</dt>
+<dd>An event that wins the election to be made a judge. It must be a witness, and it will have tended to have been 
+gossiped to most of the other nodes quickly (otherwise it would have lost the election). An event reaches consensus
+when it is an ancestor of all judges in a given round. The first round where that happens is its consensus round. 
+It’s a math theorem that every round is guaranteed to have at least one judge, and a math conjecture that every round
+is guaranteed to have judges created by a supermajority of nodes (>2/3 of weight).</dd>
+<dt>Active Node</dt>
+<dd>A node is an *Active Node* if it creates a "judge" in a significant fraction of rounds during a staking period, 
+for example 80%. Any node that is trustworthy and submits events will create judges. If the node doesn’t submit events
+or when other nodes don’t build on this node’s events, the node will not create judges.</dd>
+</dl>
+
+## Motivation
+
+The goal is to give a strong incentive for node operators to keep them active and properly configured throughout every
+staking period.
+
+## Rationale
+
+Clearly defining the criteria for node activity, we can ensure that rewards are distributed fairly, incentivizing nodes
+to remain active and contribute to the network’s stability and growth.
+
+## User Stories
+
+The implementation will be tested under various scenarios, including:
+
+- As a node operator, I expect to receive at least a minimum node reward when the node is active for at least
+  `nodes.activeRoundsPercent` rounds
+- As a node operator, I do not expect to receive any node rewards when the node is not active for at least
```

Comment:
I understand this is following the usual pattern for use cases, but I find it weird nevertheless. How about something like:

"As a user of the network, I expect that non-active nodes do not receive any node rewards..."

Reply:
> As a user of the network

Do you mean the node operator Michael?

Reply:
No, I meant any regular user. It sounds weird for me to have a use case saying that somebody does not expect a certain behavior. However, as a regular user, I expect that the money I spend is only used for active nodes.

It's not important, though. It is clear what the expected behavior is.

Reply:
Ah makes sense. Thanks @netopyr . Will add another use case